src/app/pages/categories/categories.component.html

```diff
@@ -2,7 +2,7 @@
   <div class="categories-category" *ngFor="let category of categories">
     <a class="categories-header" [id]="category.slug + '-header'" (click)="navigate(category)">
       <h2 [id]="category.slug + '-title'">{{category.name}}</h2>
-      <p routerLink="/categories/{{category.id}}">
+      <p (click)="navigate(category)">
```

Comment:
All the card is clickable, you don't need to add this click option

Reply:
I need to because if you put your cursor on "Voir plus" and click on it, sometimes it doesn't work so I put it there to avoid that